### DISCUSSION TITLE
v2.0.0 Technical Preview 2

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#1998] [分享 / Share]
- 设置面板更新:
  - 支持右侧停靠
  - 修复组件列表溢出面板
  - 动画效果调整
- 更新了动态 API (`src/components/feeds/api`)
- 组件迁移:
  - 禁止跳转动态详情
  - 直播信息扩充
  - 快速收起评论区
  - 展开动态标题
  - 动态反折叠
  - 删除直播水印
  - 专栏文字选择
  - 网址参数清理
  - 稍后再看重定向
  - 启用弹幕空降
  - 跳过充电鸣谢
- 自定义顶栏中的搜索改为默认不显示推荐词
- 样式输入格式改为与组件和插件一致 (使用 `.js` 文件), 修复样式无法卸载

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.0-tp2'>v2.0.0 Technical Preview 2</a>.</em>

**Comment by the1812** [comment]
第二批迁移功能计划:

- [x] 动态
  - [x] 展开动态内容
- [x] 视频
  - [ ] ~~启用逐帧调整~~ 3.0播放器依然没有灰度到我, 先搁置了
  - [x] 外置稍后再看
  - [x] 启用快速收藏
  - [x] 自动展开视频简介
  - [x] 扩展视频倍速
  - [x] 删除关联视频弹窗 (以及另外几种弹窗)
  - [x] 快捷键扩展
- [x] 直播
  - [x] 勋章快速切换
  - [x] 缩放看板娘
  - [x] 收起直播侧栏
  - [x] 简化直播间

**Comment by RingingVolcano** [comment]
肝 the佬 肝
